owl-zoo.opam

```diff
@@ -26,4 +26,5 @@ depends: [
   "dune" {>= "1.7.0"}
   "ocaml-compiler-libs"
   "owl" {>= version}
+  "npy"
```

Comment:
same here, I think owl.opam already specifies this dependency.

Reply:
ar okay I wasn't sure. will remove.